README.md

```diff
@@ -302,24 +315,30 @@ With specific system environment variables you can alter the behaviour of the CL
 
 # Development
 
-After cloning the project, install dependencies with:
+After the project has been cloned, the dependencies must be
+installed. Run the following in the project folder:
 
 ```sh
- $ npm i
+ $ npm install
 ```
 
-In order to compile NodeJS code run
+Then you need to compile the TypeScript code:
 
 ```sh
  $ npm run compile
 ```
 
-and you can try out the `swarm-cli` CLI after run command
+To install `swarm-cli` as a global package:
```

Comment:
it won't install anything, only creates a symlink to the global npm bins folder that points to the source code's dist folder

Reply:
i've reworded that part